src/I18n/MessagesFileLoader.php

```diff
@@ -126,7 +125,7 @@ public function __invoke()
         }
 
         if (!$file) {
-            return $package;
+            return false;
```

Comment:
Yes. I need some way to differentiate when the file doesn't exists or doesn't have any translations.